No such property: sdkDirectory for class: com.android.build.gradle.AppPlugin

Just updated to 0.10.0 of the build tools, and now `./gradlew spoon` fails with "No such property: sdkDirectory for class: com.android.build.gradle.AppPlugin"

Stacktrace...

```
Caused by: groovy.lang.MissingPropertyException: No such property: sdkDirectory for class: com.android.build.gradle.AppPlugin
at com.stanfy.spoon.gradle.SpoonRunTask.runSpoon(SpoonRunTask.groovy:110)
at org.gradle.internal.reflect.JavaMethod.invoke(JavaMethod.java:63)
at org.gradle.api.internal.project.taskfactory.AnnotationProcessingTaskFactory$StandardTaskAction.doExecute(AnnotationProcessingTaskFactory.java:219)
at org.gradle.api.internal.project.taskfactory.AnnotationProcessingTaskFactory$StandardTaskAction.execute(AnnotationProcessingTaskFactory.java:212)
at org.gradle.api.internal.project.taskfactory.AnnotationProcessingTaskFactory$StandardTaskAction.execute(AnnotationProcessingTaskFactory.java:201)
at org.gradle.api.internal.AbstractTask$TaskActionWrapper.execute(AbstractTask.java:533)
at org.gradle.api.internal.AbstractTask$TaskActionWrapper.execute(AbstractTask.java:516)
at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeAction(ExecuteActionsTaskExecuter.java:80)
at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeActions(ExecuteActionsTaskExecuter.java:61)
```

--- a/example/build.gradle
+++ b/example/build.gradle
@@ -5,7 +5,7 @@ buildscript {
   }
 
   dependencies {
-    classpath 'com.android.tools.build:gradle:0.9.2'
+    classpath 'com.android.tools.build:gradle:0.10.0'
     classpath 'com.stanfy.spoon:spoon-gradle-plugin:0.9.7-SNAPSHOT'
   }
 }

--- a/src/main/groovy/com/stanfy/spoon/gradle/SpoonRunTask.groovy
+++ b/src/main/groovy/com/stanfy/spoon/gradle/SpoonRunTask.groovy
@@ -107,7 +107,7 @@ class SpoonRunTask extends DefaultTask implements VerificationTask {
         .setDebug(debug)
         .setClassName(className)
         .setMethodName(methodName)
-        .setAndroidSdk(project.plugins.withType(AppPlugin).toList().get(0).sdkDirectory)
+        .setAndroidSdk(project.android.sdkDirectory)
         .setClasspath(cp)
         .setNoAnimations(noAnimations)
 

